src/libraries/System.Memory.Data/src/System/BinaryData.cs

```diff
@@ -323,13 +436,7 @@ public override unsafe string ToString()
         /// </summary>
         /// <param name="data">The value to be converted.</param>
         public static implicit operator ReadOnlySpan<byte>(BinaryData? data)
-        {
-            if (data == null)
-            {
-                return default;
-            }
-            return data._bytes.Span;
-        }
+            => data is null ? default : data._bytes.Span;
```

Comment:
Please refrain from making pure style changes like this.  It's unrelated to the purpose of the PR.